mesh/meshdb.go

```diff
@@ -399,9 +399,14 @@ type dbReward struct {
 }
 
 func (m *MeshDB) writeTransactionRewards(l types.LayerID, accounts []types.Address, totalReward, layerReward *big.Int) error {
-	batch := m.transactions.NewBatch()
+	actBlockCnt := make(map[types.Address]uint64)
 	for _, account := range accounts {
-		reward := dbReward{TotalReward: totalReward.Uint64(), LayerRewardEstimate: layerReward.Uint64()}
+		actBlockCnt[account]++
```

Comment:
does `accounts` actually contain the same account twice if reward needs to be paid twice?

Reply:
Yes.